### DISCUSSION TITLE
Sacrifice layers for sample rate?

### DISCUSSION THREAD

**Discussion by Worlingham** [probonopd/MiniDexed#415] [General]
Hi all, great work by all contributors. 

Was wondering if there was a way to sacrifice multitimbralaty(?) for a higher sample rate ?

**Comment by probonopd** [comment]
Hi @Worlingham. Quick research shows that the DX7 was using a 49096Hz sample rate. MiniDexed is using `SampleRate=48000` in `minidexed.ini` by default, which is close. What sample rate are you after? Have you tried to set it in `minidexed.ini` and what happened?

**Reply by Worlingham** [reply]
Hi! Sorry I didn't write very clearly, basically I would like to run this at 96khz but am using rpi 2


 so to preserve some processing power it would be good if I could disable all the other tone generators / layers except for 1 ( as I think is default on the pi zero)

Was wondering if there's a way to configure this

**Reply by probonopd** [reply]
You can set the other TGs to a MIDI channel you are not using, then they don't use any processor power. But this just means that some cores will go ununsed, if I understand our current implementation correctly.

**Reply by Worlingham** [reply]
Thanks very much
